presto-main/src/main/java/io/prestosql/server/security/KerberosAuthenticator.java

```diff
@@ -207,4 +211,29 @@ private static InetAddress getLocalHost()
             throw new RuntimeException(e);
         }
     }
+
+    private GSSName getGSSName(KerberosConfig config, String hostname)
```

Comment:
Make these methods on the enum. See `io.prestosql.plugin.hive.benchmark.FileFormat` for an example.